src/client/reducers/projects.ts

```diff
@@ -58,6 +77,42 @@ const projectsReducer: LoopReducer<ProjectsState, Action> = (
         },
         Cmd.run(showResourceFailedToast)
       );
+      case getType(globalProjectsFetch):
+        {
+          return loop(
+            {
+              ...state,
+              globalProjects: { isPending: true }
+            },
+            Cmd.run(fetchAllPublishedProjects, {
+              successActionCreator: globalProjectsFetchSuccess,
+              failActionCreator: globalProjectsFetchFailure,
+              args: [state.globalProjectsPagination.currentPage, state.globalProjectsPagination.limit] as Parameters<typeof fetchAllPublishedProjects>
+            })
+          );
+        }
+      case getType(globalProjectsFetchPage):
+        return loop(
+          {
+            ...state,
+            globalProjectsPagination: { ...state.globalProjectsPagination , currentPage: action.payload }
+          },
+          Cmd.run(globalProjectsFetch)
+        );
```

Comment:
```suggestion
      case getType(globalProjectsSetPage):
        return {
            ...state,
            globalProjectsPagination: { ...state.globalProjectsPagination , currentPage: action.payload }
          };
```

In the current solution (where `PaginationFooter` is watching for a change in `pagination.currentPage` and then calling `globalProjectsFetch()` in `onPageChange`, this is unnecessary. However, I'm going back and forth on whether I like the current architecture more than what I was _originally_ thinking here of `globalProjectsFetchPage`. Curious your thoughts, @maurizi